source/shared/cpp/ObjectModel/HostConfig.h

```diff
@@ -407,13 +407,14 @@ struct TableConfig
 
 struct BadgeConfig
 {
-    std::string backgroundColor = "#DCECFC";
```

Comment:
There should be badge text color also? In dark mode and in light mode badge text color can be different?

Reply:
<img width="527" alt="Screenshot 2024-08-14 at 10 20 46 AM" src="https://github.com/user-attachments/assets/57e159d7-e0e8-4a55-aa66-2b4e18214e01">

Reply:
For simplicity , i am assigning badge text color to background color as in dark mode it will be black and in light mode it will be light